src/app/backend/resource/pod/list.go

```diff
@@ -123,6 +126,7 @@ func ToPodList(pods []v1.Pod, events []v1.Event, nonCriticalErrors []error, dsQu
 		warnings := event.GetPodsEventWarnings(events, []v1.Pod{pod})
 		podDetail := ToPod(&pod, metrics, warnings)
 		podDetail.Warnings = warnings
+		podDetail.NodeName = pod.Spec.NodeName
```

Comment:
I know. It is done like this in a few places. Didn't touch it now, as this is not part of pull request, but I can change it if you want.